### DISCUSSION TITLE
Add global search bar

### DISCUSSION THREAD

**Discussion by bannarisoftwares** [arthelokyo/astrowind#503] [Ideas]
Add global search like this astro template https://starlight.astro.build/

**Comment by prototypa** [comment]
Hi @bannarisoftwares 

It's certainly a nice feature.

We are going to review how it is implemented in _Starlight_ template and if it is relatively simple, we will develop the search functionality in **AstroWind**.

Let's move this issue to `Discussions/Ideas`.

**Reply by twentytwokhz** [reply]
Any updates on this? I believe this is the **only** feature remaining that is keeping this repo from being stellar!
